src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * {@code Deadline} is a type of {@code Task} where user must specify the description and the "/by" remark
+ */
+public class Deadline extends Task {
+
```

Comment:
Perhaps you want to rename Deadline to DeadlineTask to make it obvious that it is a Task?